chain-crypto/src/algorithms/mod.rs

```diff
@@ -13,4 +12,4 @@ pub use ed25519::Ed25519;
 pub use ed25519_derive::Ed25519Bip32;
 pub use ed25519_extended::Ed25519Extended;
 pub use sumed25519::SumEd25519_12;
-pub use vrf::EllipticCurve2hashDh;
+pub use vrf::Curve25519_2HashDh;
```

Comment:
Right, can you deuglify the names while we are at it?
Maybe a submodule namespace like `vrf::curve25519::Curve2HashDh` could help.

Reply:
I would use `ristretto` instead of `curve25519`, as it is more representative of what we use.